### ISSUE TITLE
Remove Joe's write access?

### ISSUE THREAD

**Issue by matthewcarbone** [AI-multimodal/aimmdb#55]
@danielballan @jmaruland we should probably remove Joe's generic [write access](https://github.com/AI-multimodal/aimmdb/blob/69238c66438230b36c76c93242b2062764839f8b/deploy/spin/config/config.yml) if only as a precaution. I can easily do this but I don't want to break anything inadvertently.

**Comment by x94carbone** [comment]
@danielballan any thoughts?

**Comment by danielballan** [comment]
Yes, that's fine. On a related note, I just changed Joe's role in the AI-multimodal GitHub org from "Owner" to "Member".

This prompts a follow-up question on where the group truth configuration file is stored. @jmaruland mentioned, I think, that his process is to hand-edit a file on the CFS via `cori`. Is the file in version control in sync with that file?
